extensions/standard-processors/tests/unit/GetFileTests.cpp

```diff
@@ -292,3 +292,26 @@ TEST_CASE("GetFile sets attributes correctly") {
     CHECK(flow_file->getAttribute(minifi::core::SpecialFlowAttribute::FILENAME) == "beta.txt");
   }
 }
+
+TEST_CASE("GetFile can use expression language in Directory property") {
```

Comment:
Based on the latest windows test failure you need something like here, and you would also need to define TZ_DATA_DIR in extensions/standard-processors/tests/CMakeLists.txt for unit tests aswell (currently its only defined for integration tests)
```suggestion
TEST_CASE("GetFile can use expression language in Directory property") {
#ifdef WIN32
  expression::dateSetInstall(TZ_DATA_DIR);
#endif
```

Reply:
Thanks! Updated in 70c7209706b5acda7b95051f57fad8168bdd7712